### PR TITLE
Clarify prerelease package documentation

### PR DESCRIPTION
**Pull request by loic-sharma**
Addresses https://github.com/NuGet/docs.microsoft.com-nuget/issues/1423

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6dc0826](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1424/commits/6dc082618ebc43da3b260f4de43f2ca8273868ba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-11/docs/create-packages/Prerelease-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1424) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904292011392505-1424%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZJYwdDanOQ9GMtDxHFZUtxm1yzdYTMRdgyiuhrgC5UA%3D&st=2019-04-29T20%3A08%3A01Z&se=2019-05-30T20%3A13%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[268a02f](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1424/commits/268a02fcfc066e087047b5b60d6d98c5294ac7c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-11/docs/create-packages/Prerelease-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1424) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904292021506787-1424%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZORwVSnlMWDHRBHhVpWUwlYbLlteBEdmq7eRKmgg7GE%3D&st=2019-04-29T20%3A18%3A10Z&se=2019-05-30T20%3A23%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Prerelease-Packages.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ A stable release is one that's considered reliable enough to be used in producti
...
 
-- `.nuspec` file: include the semantic version suffix in the `version` element:
+- [`nuget.exe`](../tools/nuget-exe-cli-reference.md): include the semantic version suffix in the `.nuspec` file's `version` element:
 
     ```xml
     <version>1.0.1-alpha</version>
     ```
 
-- `.csproj` file: include the semantic version suffix in the `PackageVersion` element:
+- [dotnet CLI](/dotnet/articles/core/tools/index): include the semantic version suffix in the `.csproj` file's  `PackageVersion` element:
```
Comment (nkolev92) [inline docs/create-packages/Prerelease-Packages.md]: I think we should refrain from using dotnet CLI as a feature descriptor. 

We prefer csproj with further clarification wherever possible.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[d4eb4e9](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1424/commits/d4eb4e97afdac35a626ff578bd3f999d9189bcec)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-11/docs/create-packages/Prerelease-Packages.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1424) | [Details](#user-content-6e6c93e7-3d20fca7)

<a id="6e6c93e7-3d20fca7"></a>
### [docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-11/docs/create-packages/Prerelease-Packages.md)
 - **Line 22**: **[Warning]** Invalid file link:(~/consume-packages/package-references-in-project-files).
 - **Line 30**: **[Warning]** Invalid file link:(~/reference/packages-config).
 - **Line 30**: **[Warning]** Invalid file link:(~/reference/nuspec).
 - **Line 30**: **[Warning]** Invalid file link:(~/reference/nuspec#version).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904292122150541-1424%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SEh3Ke5xETnmZ77DsIgpMmYLZrOKBvMBRjxcl8%2BZrN8%3D&st=2019-04-29T21%3A18%3A31Z&se=2019-05-30T21%3A23%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[0b1dca1](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1424/commits/0b1dca14d1333d7c183a66c7c12d861260a26c27)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-11/docs/create-packages/Prerelease-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1424) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904292127109990-1424%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Bgo2%2BBLdawXYA2oYeQ84eKdYMd6NRoVw81VKN3A0odY%3D&st=2019-04-29T21%3A23%3A34Z&se=2019-05-30T21%3A28%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by loic-sharma** [comment]
@nkolev92, could you review again please? :)

**Pull request merged**